src/views/templates/actions/hooks/useVirtualMachineTemplatesActions.tsx

```diff
@@ -81,9 +81,19 @@ const useVirtualMachineTemplatesActions: useVirtualMachineTemplatesActionsProps
     },
     {
       id: 'edit-boot-source',
+      label: t('Edit boot source'),
+      description: isCommonTemplate && t('Red Hat template cannot be edited'),
+      disabled: isCommonTemplate,
+      cta: () =>
+        history.push(
+          `/k8s/ns/${template.metadata.namespace}/templates/${template.metadata.name}/disks`,
+        ),
+    },
+    {
+      id: 'edit-boot-source-ref',
       label: (
         <>
-          {t('Edit boot source')} {editableBootSource === null && <Loading />}
+          {t('Edit boot source reference')} {editableBootSource === null && <Loading />}
```

Comment:
i think editableBootSource === null can be replaced with !editableBootSource, please check

Reply:
Thanks, Matan :-). `editableBootSource` starts with null (not defined) and after a request, it can become true or false. Maybe I can use the `??` operator.
Currently, in another pr I'm going to edit this logic so let me know if this logic is a little tricky